test/sanity/ignore.txt

```diff
@@ -3693,46 +3693,10 @@ lib/ansible/modules/network/fortios/fortios_firewall_sniffer.py validate-modules
 lib/ansible/modules/network/fortios/fortios_ipv4_policy.py validate-modules:E337
 lib/ansible/modules/network/fortios/fortios_ipv4_policy.py validate-modules:E338
 lib/ansible/modules/network/fortios/fortios_report_chart.py validate-modules:E326
-lib/ansible/modules/network/fortios/fortios_spamfilter_profile.py validate-modules:E336
-lib/ansible/modules/network/fortios/fortios_spamfilter_profile.py validate-modules:E337
-lib/ansible/modules/network/fortios/fortios_ssh_filter_profile.py validate-modules:E336
-lib/ansible/modules/network/fortios/fortios_ssh_filter_profile.py validate-modules:E337
-lib/ansible/modules/network/fortios/fortios_switch_controller_global.py validate-modules:E336
-lib/ansible/modules/network/fortios/fortios_switch_controller_global.py validate-modules:E337
 lib/ansible/modules/network/fortios/fortios_switch_controller_lldp_profile.py validate-modules:E336
```

Comment:
We should not be ignoring this check. Some attribute is not a valid Python Identifier.